integration-tests/bootstrap/bootstrap_test.go

```diff
@@ -0,0 +1,18 @@
+package bootstrap
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Getting Kubernetes version", func() {
+	Context("using kubeconfig", func() {
+		It("should return the version of kubernetes", func() {
+			KubernetesVersion, err := kubeConfig.Discovery().ServerVersion()
+			Expect(err).To(BeNil())
```

Comment:
Put verifications on versions like version is not empty & so on

Reply:
done